OpenRA.Game/Network/GameServer.cs

```diff
@@ -226,6 +229,7 @@ public GameServer(Server.Server server)
 			Protected = !string.IsNullOrEmpty(server.Settings.Password);
 			Authentication = server.Settings.RequireAuthentication || server.Settings.ProfileIDWhitelist.Any();
 			Clients = server.LobbyInfo.Clients.Select(c => new GameClient(c)).ToArray();
+			DisabledSpawnPoints = server.LobbyInfo.DisabledSpawnPoints;
```

Comment:
We will need to clone the list to prevent server changes from mutating this record of the state. Use `.ToList()` or, better (because it is more immutable), define `DisabledSpawnPoints` as an array and use `.ToArray()`